feature-libs/my-account/organization/occ/converters/occ-cost-center-normalizer.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { CostCenter, Converter, Occ } from '@spartacus/core';
+
+@Injectable()
+export class OccCostCenterNormalizer
```

Comment:
We have similar normalizer for cost center in the @spartacus/core from B2B Checkout